web/hand_tool.js

```diff
@@ -46,12 +43,12 @@ var HandTool = (function HandToolClosure() {
 
     this.eventBus.on('togglehandtool', this.toggle.bind(this));
 
-    Promise.all([localized,
-                 preferences.get('enableHandToolOnLoad')]).then((values) => {
+    let enableOnLoad = preferences.get('enableHandToolOnLoad');
+    Promise.all([localized, enableOnLoad]).then((values) => {
```

Comment:
I had trouble with `([, value]) =>` with construct. Let's keep it as is -- I have something in mind to remove this statement later.